ports/raspberrypi/boards/adafruit_feather_rp2040_usb_host/mpconfigboard.h

```diff
@@ -12,3 +12,8 @@
 
 #define DEFAULT_UART_BUS_RX (&pin_GPIO1)
 #define DEFAULT_UART_BUS_TX (&pin_GPIO0)
+
+#define CIRCUITPY_CONSOLE_UART_RX DEFAULT_UART_BUS_RX
+#define CIRCUITPY_CONSOLE_UART_TX DEFAULT_UART_BUS_TX
```

Comment:
should this remain or was it for debugging?

Reply:
Debugging. Commented it out.